src/style/design-tokens/theme-compatibility.util.js

```diff
@@ -21,6 +21,7 @@ export default (theme) => ({
   colorsYin055: "rgba(0,0,0,0.55)",
   colorsYin065: "rgba(0,0,0,0.65)",
   colorsYin090: "rgba(0,0,0,0.9)",
+  colorsYang100: "rgba(255,255,255,1)",
```

Comment:
`theme-compatibility.util` is eventually removed. This has to be moved to `compatibility` section in theme-configs.